File: C#/5namesandages.cs
//Write a program that has a dictionary of 3 people with their name and age. Print each person in this format:
//[name] is [age] years old.
using System;
using System.Collections.Generic;

class Geeks
{
    public static void Main()
    {
        Dictionary<string, int> names = new Dictionary<string, int>();
		names.Add("Peter", 35);
		names.Add("Ben", 37);
		names.Add("Joe", 43);
		
		foreach(var ele in names){
			Console.WriteLine($"{ele.Key} is {ele.Value} years old");
		}
    }
}

File: java/5namesandages.java
import java.util.HashMap;
public class Main {
  public static void main(String[] args) {
    //Write a program that has a HashMap of 3 people with their name and age. Print each person in this format:

//[name] is [age] years old.
HashMap<String, Integer>  names = new HashMap <String, Integer>();
names.put("Peter", 35);
names.put("Ben", 37);
names.put("Joe", 43);
for (String name : names.keySet()){
	
    System.out.println(name + " is "+ names.get(name)+ " years old.");
    }
    
  }
}
